ui/components/requirements/requirement.jsx

```diff
@@ -15,13 +16,37 @@ export default function Requirement({ requirement }) {
       <div className="req-text col col-10">
         { reqTexts }
         <div className="clearfix mt3">
-          <div className="applies-to mr2">
+          <div className="applies-to mr2 metadata">
             Applies to: [not implemented]
           </div>
-          <div className="sunset-date">
-            Sunset date by { requirement.policy.sunset || 'none' }
+          { requirement.issuing_body ? (
+            <div className="issuing-body metadata">
+              Issuing body: { requirement.issuing_body }
+            </div>
+          ) : (
+            ''
+          ) }
```

Comment:
Just going to note this here: `key` turns out to be some kind of secretly-hidden reserved word—if you try code similar to the above using an argument named `key` its value will show up in the output as `undefined`. If you change `key` to `name` it all works fine.

(This is true on my machine, anyway…)